tripper/backends/rdflib.py

```diff
@@ -115,13 +115,32 @@ def serialize(
             return result if isinstance(result, str) else result.decode()
         return None
 
-    def query(self, query_object, **kwargs):
-        """SPARQL query."""
-        # TODO: convert to returned object
-        return self.graph.query(query_object=query_object, **kwargs)
+    def query(self, query_object, **kwargs) -> "List[Tuple[str, ...]]":
+        """SPARQL query.
 
-    def update(self, update_object, **kwargs):
-        """Update triplestore with SPARQL."""
+        Parameters:
+            query_object: String with the SPARQL query.
+            kwargs: Keyword arguments passed to rdflib.Graph.query().
+
+        Returns:
+            List of tuples of IRIs for each matching row.
+
+        """
+        rows = self.graph.query(query_object=query_object, **kwargs)
+        return [tuple(str(v) for v in row) for row in rows]
```

Comment:
```suggestion
        return [
            tuple(str(v) for v in row)
            for row in self.graph.query(query_object=query_object, **kwargs)
        ]
```

Reply:
What is the benefit of this change? I feel that the original code is slightly easier to read and two lines shorter.

Reply:
I found this more readable, hence the suggested change. I have no issues if you prefer the other way - I just prefer not making variables unnecessarily just to make variables, and reading directly where each row is taken from seems more readable to me. But again, it's just a suggested change, not a necessary one.